docs/src/SUMMARY.md

```diff
@@ -19,6 +19,7 @@
 * [Testing Contracts' Internals](testing/testing_contract_internals.md)
 * [Using Cheatcodes](testing/using-cheatcodes.md)
 * [Testing Events](testing/testing-events.md)
+* [Test Attributes](testing/test-attributes.md)
```

Comment:
nit: Given that this is quite basic information, wouldn't it be better to put it higher as point `7.` right after the section `Writing Tests`?